.circleci/config.yml

```diff
@@ -10,23 +10,27 @@ commands:
       - run: sudo apt-get install python3-pip arping ndisc6
       - run: sudo pip3 install invoke semver pyyaml
 jobs:
-  test-1-16:
+  test-1-17:
     machine:
       image: ubuntu-2004:202104-01
     steps:
+      - run: |
+          curl -o /tmp/go1.17.1.linux-amd64.tar.gz https://dl.google.com/go/go1.17.1.linux-amd64.tar.gz
+          sudo rm -rf /usr/local/go
+          sudo tar -C /usr/local -xzf /tmp/go1.17.1.linux-amd64.tar.gz
```

Comment:
how come with go 1.16 we didn't have to curl?

Reply:
the issue here is, we are using a vm (as opposed to the docker image for the other lanes) because frr tests are using docker. The version of go that came with it was old and not compatible with our codebase (some of the dependencies, actually).